src/folio_migration_tools/migration_tasks/requests_migrator.py

```diff
@@ -40,6 +40,23 @@ def __init__(
         task_configuration: TaskConfiguration,
         library_config: LibraryConfiguration,
     ):
+        try:
+            logging.info(
+                "Attempting to retrieve tenant timezone configuration..."
+            )
+            self.tenant_timezone_str = json.loads(self.folio_client.folio_get_single_object(
```

Comment:
I do not think you need the json.loads here. I wonder if Python makes it work by taking the dict returned from the method and transforms it to a string

Reply:
This falls back on me though, that is likely not using type hints in FolioClient

Reply:
The response from FOLIO actually contains escaped JSON in the JSON response. I want to run a couple more tests today before I merge.

Reply:
It's quoted/escaped json within the returned json object.